### PR TITLE
Improve performance of expandIP, and readable

### PR DESCRIPTION
**Pull request by 844392610**

### CODE REVIEW

**Comment by ltamber** [comment]
It would be better if you can submit an issue related with this pull request

File: acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java [hunk 1]
```diff
@@ -206,61 +206,31 @@ public static boolean isIPv6Scope(int num) {
...
-            if (netaddress.charAt(i) == ':') {
-                if (indexes.size() > 0 && i - indexes.get(indexes.size() - 1) == 1) {
-                    compressIndex = i;
-                    compress = true;
-                }
-                indexes.add(i);
+        // expand netaddress
+        int separatorCount = StringUtils.countMatches(netaddress, ":");
+        int padCount = part - separatorCount;
+        if(padCount > 0){
```
Comment (yuz10) [inline acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java:212]: make sure the checkstyle pass. run `mvn -Prelease-all -DskipTests install -U` to check build

File: acl/src/test/java/org/apache/rocketmq/acl/common/AclUtilsTest.java [hunk 2]
```diff
@@ -31,6 +31,8 @@
 
 public class AclUtilsTest {
 
+    //  strArray[i] = StringUtils.leftPad(strArray[i],4, '0');
```
Comment (yuz10) [inline acl/src/test/java/org/apache/rocketmq/acl/common/AclUtilsTest.java:34]: The comment is meaningless

**Review by yuz10 — CHANGES_REQUESTED** [review]

**Comment by yuz10** [comment]
You should merge into develop branch , not master

**Comment by 844392610** [comment]
move to #3319

**Pull request closed without merging**